### DISCUSSION TITLE
Benchmark for frameworks using sqlite

### DISCUSSION THREAD

**Discussion by anilbhattaraitoronto** [TechEmpower/FrameworkBenchmarks#6515] [Ideas]
I am wondering if there has been any bench marking of frameworks using sqlite as database engine? From sqlite's documentations and ongoing discussions, it seems sqlite is pretty good for small to medium sized web applications and, maybe it will be nice to see some benchmarks. I am new to programming and hence do not possess the competence to perform this task, I am really keen if bench marks exist for sqlite integrated frameworks

Thank you

**Comment by NateBrady23** [comment]
We don't currently support sqlite but it's something that can probably be added pretty easily in the future.

**Comment by joanhey** [comment]
SQLite is file-based only. There is no way to talk to it over TCP/IP.
